test-suite/buildtest.sh

```diff
@@ -16,6 +16,18 @@
 action="${1}"
 config="${2}"
 
+# Allow a layer to enable optional default-disabled features when
+# those features are supported in the current build environment
+# (and we can easily detect such support).
+if ${PKG_CONFIG:-pkg-config} --exists --exact-version=1.0.0 libecap 2>/dev/null
```

Comment:
Our configure.ac says `[libecap >= 1.0 libecap < 1.1]`.

The test suggested in this PR seems to exclude, say, v1.0.1 that would have been included by ./configure. If possible, please match that (more permissive) ./configure condition.

N.B. PR-added tests should not depend on whether a particular package version exists _today_.

Reply:
Done (a0f0131).